### PR TITLE
docs: v2.0 content updates

### PR DESCRIPTION
**Pull request by ryanchristo**
## Description

Closes: #566

This pull request includes general updates to the documentation to reflect the latest information.

- Update getting started to use v2.0
- #566
- Add node configuration instructions
- Update core content to reflect v2.0
- Update v2.0 migration documentation
  - Add Redwood block height
  - Update cosmovisor warning

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [x] provided a link to the relevant issue or specification
- [x] followed the [documentation writing guidelines](https://github.com/cosmos/cosmos-sdk/blob/master/docs/DOC_WRITING_GUIDELINES.md)
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct `docs:` prefix in the PR title
- [ ] confirmed all author checklist items have been addressed 
- [ ] confirmed that this PR only changes documentation
- [ ] reviewed content for consistency
- [ ] reviewed content for thoroughness
- [ ] reviewed content for spelling and grammar
- [ ] tested instructions (if applicable)

### CODE REVIEW

**Review by dpdanpittman — APPROVED** [review]

**Review by dpdanpittman — APPROVED** [review]

File: docs/migrations/v2.0-upgrade.md [hunk 1]
```diff
@@ -31,20 +31,20 @@ We recommend node operators use [Cosmovisor](https://docs.cosmos.network/master/
...
+The upgrade to Regen Ledger v2.0 leverages a new feature introduced in Cosmos SDK v0.43 called ["In-Place Store Migrations"](https://docs.cosmos.network/master/core/upgrade.html). This new feature prevents the need for node operators to restart the chain using an updated genesis file. Regen Ledger includes a [`registerUpgradeHandlers()`](https://github.com/regen-network/regen-ledger/blob/v2.0.0/app/stable_appconfig.go#L52-L103) method that registers handlers for performing an upgrade's necessary state migrations in-place.
 
 ## Upgrade Guidelines
 
 ### Update Go
 
-We recommend updating to the latest version of Go `1.17`. For more information, see [Go](https://golang.org/).
+Please update to the latest version of Go `1.17`. For more information, see [Go](https://golang.org/).
```
Comment (ryanchristo) [inline docs/migrations/v2.0-upgrade.md:40]:
```suggestion
The latest version of Go (`1.17`) is required. For more information, see [Go](https://golang.org/).
```

File: docs/migrations/v2.0-upgrade.md [hunk 2]
```diff
@@ -31,20 +31,20 @@ We recommend node operators use [Cosmovisor](https://docs.cosmos.network/master/
...
 
 ### Option 1: Using Cosmovisor
 
-We recommend **Cosmovisor v1.0 with the auto-download option disabled**. Node operators will need to prepare the upgrade binary in advance.
+Please use **Cosmovisor v1.0 with the auto-download option disabled**. Node operators must prepare the upgrade binary in advance.
```
Comment (ryanchristo) [inline docs/migrations/v2.0-upgrade.md:44]:
```suggestion
**Cosmovisor v1.0 with the auto-download option disabled** is required. Node operators must prepare the upgrade binary in advance.
```

File: docs/getting-started/live-networks.md [hunk 3]
```diff
@@ -10,10 +10,10 @@ By default, the `regen` binary will connect to a local node. In order to connect
 
 `regen-1` is the chain ID for Regen Mainnet.
 
-<!-- TODO: add information about genesis binary and upgrade binaries -->
-
 Regen Mainnet launched with the `v1.0.0` release tag of `regen-ledger`.
 
+When starting a full node or a validator node from genesis, you will need to start the node with the `v1.0.0` version (the "genesis binary"). For more information about preparing your node to migrate to the `v2.0.0` version, see [Upgrade Guide v2.0](../migrations/v2.0-upgrade.md).
```
Comment (clevinson) [inline docs/getting-started/live-networks.md:15]: We should at some point follow this up with instructions to use [state sync](https://blog.cosmos.network/cosmos-sdk-state-sync-guide-99e4cf43be2f) instead.

File: docs/getting-started/live-networks.md [hunk 4]
```diff
@@ -26,10 +26,10 @@ For more information, see [regen-network/mainnet](https://github.com/regen-netwo
 
 `regen-redwood-1` is the chain ID for Redwood Testnet.
 
-<!-- TODO: add information about genesis binary and upgrade binaries -->
-
 Redwood Testnet launched with the `v1.0.0` release tag of `regen-ledger`.
 
+When starting a full node or a validator node from genesis, you will need to start the node with the `v1.0.0` version (the "genesis binary"). For more information about preparing your node to migrate to the `v2.0.0` version, see [Upgrade Guide v2.0](../migrations/v2.0-upgrade.md).
```
Comment (clevinson) [inline docs/getting-started/live-networks.md:31]: ditto

**Review by clevinson — APPROVED** [review]

File: docs/getting-started/live-networks.md (hunk 3 above)
Comment (clevinson) [reply docs/getting-started/live-networks.md:15]: Just wrote this up in #625 so we don't forget

**Pull request merged**